src/components/FlippableHandler.js

```diff
@@ -8,6 +8,16 @@ class FlippableHandler {
         if (!FlippableHandler.flippableHandlerInitialised) {
             /** @type {HTMLElement} */
             const $rotationContainer = (document.getElementById('rotation-container'));
+            if (window.location.hash) {
```

Comment:
This is actually not relevant for PasswordBox, but is a fix for the recent export PR.